dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -279,19 +281,15 @@ abstract class DLCWallet
     val announcements =
       contractInfo.oracleInfos.head.singleOracleInfos.map(_.announcement)
 
-    //hack for now to get around https://github.com/bitcoin-s/bitcoin-s/issues/3127
-    //filter announcements that we already have in the db
-    val groupedAnnouncementsF: Future[AnnouncementGrouping] = {
-      groupByExistingAnnouncements(announcements)
-    }
-
     val feeRateF = determineFeeRate(feeRateOpt).map { fee =>
       SatoshisPerVirtualByte(fee.currencyUnit)
     }
 
     for {
       feeRate <- feeRateF
-      groupedAnnouncements <- groupedAnnouncementsF
+      //hack for now to get around https://github.com/bitcoin-s/bitcoin-s/issues/3127
+      //filter announcements that we already have in the db
+      groupedAnnouncements <- groupByExistingAnnouncements(announcements)
```

Comment:
why move this?

Reply:
For brevity and clarity: one line > four lines